libs/sdk-ui-vis-commons/src/legend/helpers.ts

```diff
@@ -352,6 +381,12 @@ function getHorizontalShorteningLabelConfig(
 ): IColorLabelConfigItem[] {
     const shorteningLevel = getColorLabelShorteningLevel(labelLengths, maxLabelLength);
     if (size === "small") {
+        /**
+         * Geo chart color legend has only six 7 labels
```

Comment:
Detail: I do not understand the comment, can you please make it clearer? I am confused with the "six 7 labels" part

Reply:
My bad...I wanted to write six but then I realized that it is seven but so I put 7 and did not remove six :D Good catch, I will fix it

Reply:
Done https://github.com/gooddata/gooddata-ui-sdk/pull/1341/commits/5e070a8c937eca1610fda9434a086420b4dcc432